src/vorta/utils.py

```diff
@@ -416,14 +417,27 @@ def format_archive_name(profile, archive_name_tpl):
     """
     hostname = socket.gethostname()
     hostname = hostname.split(".")[0]
+    fqdn = _getfqdn(hostname)
+    borg_version = os.getenv("BORG_VERSION")
+    if not borg_version:
+        borg_version = subprocess.run(['borg', '--version'], stdout=subprocess.PIPE).stdout.decode('utf-8')
```

Comment:
You are basically sidestepping everything we do to correctly detect the right `borg` binary. So while this may work in many cases, it will break in others. So as already mentioned before, it will be better to use the existing `borg_compat` module.